common/Util.cpp

```diff
@@ -1042,6 +1043,52 @@ namespace Util
 
         return StringVector(s, std::move(tokens));
     }
+
+    int safe_atoi(const char* p, int len)
+    {
+        long ret{};
+
+        int multiplier = 1;
+        int offset = 0;
+        while (isspace(p[offset]))
+        {
+            ++offset;
+            if (offset >= len)
+            {
+                return ret;
+            }
+        }
+
+        switch (p[offset])
+        {
+            case '-':
+                multiplier = -1;
+                ++offset;
+                break;
+            case '+':
+                ++offset;
+                break;
+        }
+        if (offset >= len)
+        {
+            return multiplier * ret;
```

Comment:
This is 0, and integers don't have a concept of negative zero (and returning a negative zero for this would be wrong even if there was such a thing, because the input string is not a valid number) - why multiply here?

Reply:
You're right, that's indeed not needed, removed.